src/SemanticModels.jl

```diff
@@ -112,6 +113,9 @@ end
 include("graph.jl")
 include("definitions.jl")
 include("parse.jl")
-include("modeltools.jl")
+include("modeltools/modeltools.jl")
```

Comment:
This should be `modeltools/ModelTools.jl` by convention the entrypoint for a module is capitalized like this.